modules/dapps.js

```diff
@@ -81,21 +82,21 @@ class DApps {
 			transactionTypes.DAPP
 		] = library.logic.transaction.attachAssetType(
 			transactionTypes.DAPP,
-			new DApp(scope.db, scope.logger, scope.schema, scope.network)
+			new DApp(scope.logger, scope.schema, scope.network, scope.storage)
```

Comment:
Due to change in order of parameters, it seems in git diff as you changed all parameters, not just one.